src/index.html

```diff
@@ -15,8 +15,77 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        class="img"
+        src="/src/images/logo.png"
+        alt="Moyo_logo"
+      />
+      <nav class="nav">
+        <a
+          class="nav_link is-active"
+          href="index.html"
+        >
+          Apple
+        </a>
+        <a
+          class="nav_link"
+          href="index.html"
+        >
+          Samsung
+        </a>
+        <a
+          class="nav_link"
+          href="index.html"
+        >
+          Smartphones
+        </a>
```

Comment:
```suggestion
        <a
          class="nav_link"
          href="index.html"
        >
          Samsung
        </a>
        
        <a
          class="nav_link"
          href="index.html"
        >
          Smartphones
        </a>
```

Add spaces between multiple siblings everwhere